modules/ballerina-builtin/src/main/ballerina/ballerina/lang/strings/natives.bal

```diff
@@ -2,102 +2,7 @@ package ballerina.lang.strings;
-@doc:Param { value:"suffix: The suffix to be compared" }
-@doc:Return { value:"boolean: True if the string ends with the suffix; false otherwise" }
-public native function hasSuffix (string mainString, string suffix) (boolean);
-
-@doc:Description { value:"Returns an unescaped string by omitting the escape characters of the original string"}
-@doc:Param { value:"s: The original string argument" }
-@doc:Return { value:"string: The derived string" }
-public native function unescape (string s) (string);
-
-@doc:Description { value:"Returns a string with all the characters converted to lowercase"}
-@doc:Param { value:"s: The original string argument" }
-@doc:Return { value:"string: The string converted to lowercase" }
-public native function toLowerCase (string s) (string);
-
-@doc:Description { value:"Returns a Boolean value indicating whether a string starts with the specified prefix"}
-@doc:Param { value:"mainString: The original string argument" }
-@doc:Param { value:"prefix: The prefix to be compared" }
-@doc:Return { value:"boolean: True if the string starts with the prefix; false otherwise" }
-public native function hasPrefix (string mainString, string prefix) (boolean);
-
-@doc:Description { value:"Replaces each substring of the mainString that matches the given regular expression with the given replacement"}
-@doc:Param { value:"mainString: The original string argument" }
-@doc:Param { value:"replacePattern: The regular expression to search for" }
-@doc:Param { value:"replaceWith: The replacement string" }
-@doc:Return { value:"string: The derived string" }
-public native function replaceAll (string mainString, string replacePattern, string replaceWith) (string);
-
-@doc:Description { value:"Replaces all instances of the replacePattern string with the replaceWith string and returns the result"}
-@doc:Param { value:"mainString: The original string argument" }
-@doc:Param { value:"replacePattern: The pattern to search for" }
-@doc:Param { value:"replaceWith: The replacement string" }
-@doc:Return { value:"string: The derived string" }
-public native function replace (string mainString, string replacePattern, string replaceWith) (string);
-
-@doc:Description { value:"Splits the string with the given regular expression to produce a string array."}
-@doc:Param { value:"mainString: The original string argument" }
-@doc:Param { value:"regex: The regex to split the string" }
-@doc:Return { value:"string[]: The split string array" }
-public native function split (string mainString, string regex) (string[]);
-
-@doc:Description { value:"Converts string to a BLOB"}
-@doc:Param { value:"b: string value to be converted" }
-@doc:Param { value:"encoding: Encoding to used in the conversion" }
-@doc:Return { value:"string: BLOB representation of the given String" }
-public native function toBlob (string s, string encoding) (blob);
\ No newline at end of file
+public native function valueOf (any value) (string);
```

Comment:
What is the reason for keeping the valueOf function in ballerina.lang.Strings package?

Reply:
This is needed to removed from any type.